nltk/metrics/distance.py

```diff
@@ -202,6 +250,10 @@ def demo():
     print("Binary distance:", binary_distance(s1, s2))
     print("Jaccard distance:", jaccard_distance(s1, s2))
     print("MASI distance:", masi_distance(s1, s2))
+    print("Jaro-Winkler_distance:", jaro_winkler_distance('JONES', 'JohnSon'))
+    print("Jaro-Winkler_similarity:",
+          1-jaro_winkler_distance('JONES', 'JohnSon'))
+
```

Comment:
One more request here =)

Could you reuse the `edit_distance_exmaples` and move this bit of demo code into the loop at L240?

Reply:
Yeah sure . I was afraid of breaking the code initially, so avoided doing that